src/App.tsx

```diff
@@ -21,14 +49,14 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && <Loader />}
+              {isTodosFound && <TodoList />}
```

Comment:
The condition 'isTodosFound' will always evaluate to true because 'todos' is an array and in JavaScript, an empty array is a truthy value. Consider checking the length of 'todos' array instead.